packages/utils/src/abi_encoder/evm_data_types/method.ts

```diff
@@ -44,7 +44,19 @@ export class MethodDataType extends AbstractSetDataType {
         return returnValues;
     }
 
-    public getSignature(): string {
+    public decodeReturnValuesAsArray(returndata: string, rules?: DecodingRules): any {
```

Comment:
```suggestion
    public decodeReturnValuesAsArray(returnData: string, rules?: DecodingRules): any {
```

Always camel--case variable names

Reply:
Shouldn't the return type here be `any[]`?